src/llvmo/cscript.lisp

```diff
@@ -13,7 +13,7 @@
            #~"runtimeJit.cc"
            #~"clbindLlvmExpose.cc")
 
-(k:sources :bitcode
-           #~"intrinsics.cc"
-           #~"builtins.cc")
-
+; We probably don't need this anymore.
```

Comment:
again, if we don't support bitcode builds, we should take the plunge and outright delete this.

Reply:
Axed.